apps/cli/src/commands/run.ts

```diff
@@ -87,11 +87,18 @@ export default class Run extends Command {
         // dev file is always loaded
         const composeFiles = ["docker-compose-dev.yaml"];
 
+        // snapshot volume
+        composeFiles.push("docker-compose-snapshot-volume.yaml");
+
         // explorer
         composeFiles.push("docker-compose-explorer.yaml");
 
         // load the no-backend compose file
         if (flags["no-backend"]) {
+            const index = composeFiles.indexOf("docker-compose-snapshot-volume.yaml");
+            if (index !== -1) {
+                composeFiles.splice(index, 1);
+            }
             composeFiles.push("docker-compose-host.yaml");
         }
```

Comment:
This way looks easier to read.

```suggestion
        // explorer
        composeFiles.push("docker-compose-explorer.yaml");

        // load the no-backend compose file
        if (flags["no-backend"]) {
            composeFiles.push("docker-compose-host.yaml");
        } else {
            // snapshot volume
            composeFiles.push("docker-compose-snapshot-volume.yaml");
        }
```